src/main/java/view/OutputView.java

```diff
@@ -0,0 +1,45 @@
+package view;
+
+import domain.Car;
+import java.util.Arrays;
+import java.util.List;
+
+public class OutputView {
+    public static void printErrorMessage(String errorMessage) {
+        System.out.println(errorMessage);
+    }
+
+    public static void printHeadLine() {
+        System.out.println();
+        System.out.println("실행 결과");
+    }
+
+    // TODO : 대쉬 반복 ..잘해보기
+    public static void printResult(List<Car> cars) {
+        for (Car car : cars) {
+            System.out.println(car.getCarName() + " : " + repeatDash(car.getLocation()));
```

Comment:
```suggestion
            System.out.println(String.join(" : ", car.getCarName(), "-".repeat(car.getLocation())));
```
repeatDash라는 메서드가 필요할까요??
Math.max도 필요없을것 같아요 location이 0보다 작을 수 있나요?🤔

Reply:
location 이 0보다 작을 수가 없는데 .. 코드 구현했을 때에는 뭔가 dash 를 반복하는 메서드가 좀 복잡하게 느껴져서 실수를 한 것 같네요 🥲 감사합니다!